plugins/cd/__init__.py

```diff
@@ -25,27 +25,32 @@
 # from your version.
 
 
+from six import range
```

Comment:
It should have been `from six.moves`. I'll double check other six imports too.
Is there recommended way to cover changes in plugin dirs ?